### PR TITLE
added grid

### PR DESCRIPTION
**Pull request by sashameniakin**
added grid and some updates

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,17 +11,17 @@
...
+           
     </article>
-        <article class="first_a">
+    <article>
+      <div class="div__flex">
+      <div class="div__grid">
+        <p class="p11">ME, TRAVELLING TO <br>CROATIA THIS SUMMER</p>
       <img
         class="scale"
         src="/Users/oleksandrmeniakin/Downloads/image_6487327.JPG"
```
Comment (actyralf) [inline index.html:27]: please don't use images outside of your project folder, they cannot be controlled by git/github.
a common folder structure would be:

```

/assets
   /images
      image1.jpg
      image2.jpg
   /fonts
      font1.ttf
      etc...
```

File: style.css [hunk 2]
```diff
@@ -1,5 +1,6 @@
 @import url(components/buttons.css);
```
Comment (actyralf) [inline style.css:1]: please watch the order of your `@import` directives: you should have one file **only with imports** (no css rules in it!) that controls the order. Then you should import reset / globals first and more specific files like components afterwards.

**Review by actyralf — CHANGES_REQUESTED** [review]
Well done!
I left a few comments, please take a look. I also recognized that you are using links containing only images and no text. Please use ària-label` in this case to make screen readers aware of the purpose of your links.

File: index.html (hunk 1 above)
Comment (sashameniakin) [reply index.html:27]: thank you for the comment, i will realize it in the next commit!

**Pull request merged**